core/test/create_test.jl

```diff
@@ -0,0 +1,46 @@
+@testitem "Non-positive allocation network ID" begin end
+
+@testitem "Incomplete subnetwork" begin
```

Comment:
Regarding this test: It is good practice to create your own test model to do this, but I thought of an easier way: load the parameters of an existing model, say

```julia
toml_path = normpath(@__DIR__, "../../generated_testmodels/allocation_example/ribasim.toml")
p = Ribasim.Model(toml_path).integrator.p
```

[remove an edge](https://juliagraphs.org/MetaGraphsNext.jl/dev/api/#Base.delete!-Tuple{MetaGraph,%20Any,%20Any}) to make it invalid and run your validation test on that.

Reply:
This morning I tried to load an existing model and change one of the allocation id to negative number. It gave error saying the immutable struct of type cannot be changed. I remembered you mentioned this yesterday, it probably can't be changed or maybe I didn't do it the correct way. Anyhow, in my new code I created a graph that has the required feature and used it as the test model.